### PR TITLE
Circle CI

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Enable Circle Ci

### CODE REVIEW

**Comment by StephenCoady** [comment]
@wtrocki can we look at introducing a git tagging system here? Not sure on the capabilities of CircleCI but if we could make it so that we get a tagged version of say 0.2.0 when we also mention the released aerogear versions of 0.2.0 then we have a really good way of creating a hybrid showcase/development application that SAs and developers can both use. what do you think?

**Comment by wtrocki** [comment]
@StephenCoady I actually suggested this
here: https://groups.google.com/forum/#!topic/aerogear/q_3EwNAT14M

Can you respond to this?

**Comment by StephenCoady** [comment]
@wtrocki, will do. hadn't noticed that mail.

**Comment by aliok** [comment]
Ok, build works just fine: https://circleci.com/gh/aerogear/cordova-showcase-template/4

@wtrocki wanna add this badge?
```
https://circleci.com/gh/aerogear/cordova-showcase-template.svg?&style=shield&circle-token=6977be9bf600b6b8770d784be11913497629dc8b
```

https://circleci.com/gh/aerogear/cordova-showcase-template

You can use as is. Token appended has only the permission to see the status of the build in this repo.

File: README.adoc [hunk 1]
```diff
@@ -3,9 +3,11 @@ image:https://img.shields.io/badge/iOS-Compatible-green.svg[]
 image:https://img.shields.io/badge/Ionic%20Version-3-green.svg[]
 image:https://img.shields.io/badge/Angular%20Version-4-green.svg[]
 image:https://badges.frapsoft.com/typescript/code/typescript.svg?v=101[]
+image:https://circleci.com/gh/aerogear/cordova-showcase-template.svg?&style=shield&circle-token=6977be9bf600b6b8770d784be11913497629dc8b[]
```
Comment (paolobueno) [inline README.adoc]: is it required/okay that this token's here in public?

Comment (paolobueno) [reply README.adoc]: nvm saw @aliok's comment

Comment (wtrocki) [reply README.adoc]: @paolobueno good point. It's not!

Comment (wtrocki) [reply README.adoc]: It works without that as well.

Comment (paolobueno) [reply README.adoc]: :+1: I tried the url without the token and it seemed to work too, so best not to have it

Comment (aliok) [reply README.adoc]: nope. it works for **logged in** users. 

https://circleci.com/docs/1.0/status-badges/
> (You can also link to the image without providing a circle-token, in which case the image will only appear for people who are currently logged-in to circleci.com!)

**Comment by aliok** [comment]
@wtrocki According to this: https://circleci.com/docs/1.0/status-badges/

> (You can also link to the image without providing a circle-token, in which case the image will only appear for people who are currently logged-in to circleci.com!)

**Comment by wtrocki** [comment]
@aliok Thanks for explanation. Is best to use token then.

**Pull request merged**